### ISSUE TITLE
SSO: register route - if user exists, they should be logged in with the provided email, not throw an error

### ISSUE THREAD

**Issue by Blaumaus** [Swetrix/swetrix#281]
<img width="863" alt="Screenshot_2024-02-10_at_21_46_19" src="https://github.com/Swetrix/swetrix/assets/35810911/e4107164-4a94-4e62-85f8-2e07e795e179">

**Comment by amaan-malik1** [comment]
Why don't you try to make the presign cookies?

**Comment by amaan-malik1** [comment]
Yeah why not. And what's the problem in this approach if it fix the bug? 
Could you please elaborate your bug so that I can understand it well.

**Comment by Blaumaus** [comment]
fixed in https://github.com/Swetrix/swetrix/commit/e96ff4c3dfb24bf2c455b57df93601d621691165
